packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/simplifySmalltalkCiLogCheckBoxFrame.st

```diff
@@ -0,0 +1,9 @@
+building
+simplifySmalltalkCiLogCheckBoxFrame
+	
+	^ LayoutFrame new
+		leftFraction: 0.3 offset: 0;
+		topFraction: 0.3 offset: 0;
+		rightFraction: 1 offset: 0;
+		bottomFraction: 0.3 offset: self class defaultCheckBoxHeight;
+		yourself
```

Comment:
```suggestion
building
simplifyLogCheckBoxFrame
	
	^ LayoutFrame new
		leftFraction: 0.3 offset: 0;
		topFraction: 0.3 offset: 0;
		rightFraction: 1 offset: 0;
		bottomFraction: 0.3 offset: self class defaultCheckBoxHeight;
		yourself
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code